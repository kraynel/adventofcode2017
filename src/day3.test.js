const { getDistance } = require("./day3");

describe("getDistance", () => {
  it("should compute the correct distance 1", () => {
    expect(getDistance(1)).toBe(0);
  });

  it("should compute the correct distance 2", () => {
    expect(getDistance(12)).toBe(3);
  });

  it("should compute the correct distance 3", () => {
    expect(getDistance(23)).toBe(2);
  });

  it("should compute the correct distance 4", () => {
    expect(getDistance(1024)).toBe(31);
  });

  it("should print the answer", () => {
    expect(getDistance(289326)).toMatchSnapshot();
  });
});
