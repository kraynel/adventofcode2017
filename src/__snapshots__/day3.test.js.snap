// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getDistance should print the answer 1`] = `419`;
